dist/profile/manifests/docker.pp

```diff
@@ -4,6 +4,11 @@
   class { '::docker':
   }
 
+  if ($::architecture == 'aarch64') {
+    # 'https://github.com/puppetlabs/puppetlabs-docker/issues/494
+    Apt::Source <| architecture == 'aarch64' |> { architecture => 'arm64' }
```

Comment:
fixed as of 8.4.0 apparently, I assume we're using an older version though?

Reply:
Yep: https://github.com/jenkins-infra/jenkins-infra/blob/production/Puppetfile#L34

But that one should be quite easy to bump though

Reply:
https://github.com/jenkins-infra/jenkins-infra/pull/2251

Reply:
Currently trying without this code block (removed on the last push force)

Reply:
Updated the code: the `if` condition is back with an improved comment (hoipint hat the docker module will solve issues one day..)